Add note about compiling block styles (#2)
wintercms/wn-blocks-plugin: README.md

```

Other ideas and suggestions are welcome.

> ### Note for css purge process

Reviewer: ```suggestion
### Integrating with TailwindCSS / CSS Purging
```

> If your theme uses css class purging process, it can be useful to add plugin block paths and static pages paths.
> 
> For a theme using Tailwind for example, you need to modify the tailwinf config `content` property: 

Reviewer: ```suggestion
If your theme uses CSS class purging (i.e. Tailwind), it can be useful to add the following paths to your build configuration to include the styles for any blocks defined by the theme or plugins.
```

```

Other ideas and suggestions are welcome.

### Note for css purge process

If your theme uses css class purging process, it can be useful to add plugin block paths and static pages paths.

For a theme using Tailwind for example, you need to modify the tailwinf config `content` property: 

```js

> // tailwind.config.js
> 
> const colors = require('tailwindcss/colors');
> 
> module.exports = {
>     content: [
>         './content/**/*.htm',           // Winter static pages content
>         './layouts/**/*.htm',
>         './pages/**/*.htm',
>         './partials/**/*.htm',
>         '../../plugins/winter/blocks/blocks/*.block',  // this plugin's blocks
>     ],
>     theme: {
>         ...
>     },
>     plugins: [],
> };
> 

Reviewer: ```suggestion
// tailwind.config.js
module.exports = {
    content: [
        // Winter.Pages static page content
        './content/**/*.htm',           
        './layouts/**/*.htm',
        './pages/**/*.htm',
        './partials/**/*.htm',
        './blocks/**/*.block',
        
        // Blocks provided by plugins
        '../../plugins/*/*/blocks/*.block',
    ],
};
```
